pkg/kotsclient/cluster_list.go

```diff
@@ -4,21 +4,27 @@ import (
 	"fmt"
 	"net/http"
 	"strconv"
+	"time"
 
 	"github.com/replicatedhq/replicated/pkg/types"
 )
 
-type ListClustersRequest struct {
-}
+type ListClusterRequest struct{}
 
 type ListClustersResponse struct {
 	Clusters []*types.Cluster `json:"clusters"`
 }
 
-func (c *VendorV3Client) ListClusters(includeTerminated bool) ([]*types.Cluster, error) {
-	reqBody := &ListClustersRequest{}
+type ListClustersOpts struct {
+	IncludeTerminated bool
+	StartTime         time.Time
+	EndTime           time.Time
+}
+
+func (c *VendorV3Client) ListClusters(opts ListClustersOpts) ([]*types.Cluster, error) {
+	reqBody := ListClusterRequest{}
 	clusters := ListClustersResponse{}
-	err := c.DoJSON("GET", fmt.Sprintf("/v3/clusters?include-terminated=%s", strconv.FormatBool(includeTerminated)), http.StatusOK, reqBody, &clusters)
+	err := c.DoJSON("GET", fmt.Sprintf("/v3/clusters?include-terminated=%s&start-time=%s&end-time=%s", strconv.FormatBool(opts.IncludeTerminated), strconv.FormatInt(opts.StartTime.Unix(), 10), strconv.FormatInt(opts.EndTime.Unix(), 10)), http.StatusOK, reqBody, &clusters)
```

Comment:
If `start-time` and `end-time` are optional, should this be split into 2 different requests?